pages/1.11/security/ent/secrets/unseal-store/index.md

```diff
@@ -14,25 +14,20 @@ The Secret Store can become sealed under the following circumstances.
 
 - [After being manually sealed.](/1.11/security/ent/secrets/seal-store/)
 - After a power outage.
+- Due to inability to access underlying storage (i.e. Zookeeper) by the Vault instance
 
 A sealed Secret Store cannot be accessed from the GUI. Secret values cannot be retrieved using the [Secrets API](/1.11/security/ent/secrets/secrets-api/). Services that depend on values provisioned to them via environment variables may fail to deploy.
 
-<!--
-The procedure for unsealing the Secret Store differs according to the keys used to seal it.
+To unseal the Secret Store, complete the following steps. They will unseal just a single instance of dcos-secrets. If the cluster URL obtained through `dcos config show core.dcos_url` points to a load balancer and there is more than one master instance in the cluster, then these steps should be issued against each and every master node instead and the cluster URL should be changed to the address of individual master nodes.
 
-- [Unsealing a Secret Store sealed with default keys](#unseal-def-keys).
-
-- [Unsealing a Secret Store sealed with custom keys](#unseal-cust-keys).
--->
+The intended status of the seal is persisted, so if one of the prerequisites for sealing the store occurs, dcos-secrets will unseal it again automatically even after being restarted. Only the steps depicted in [sealing the store](/1.10/security/ent/secrets/unseal-store/) will seal it back.
```

Comment:
This is a 1.11 document but links to `/1.10/security/ent/secrets/unseal-store/`, a 1.10 page.